server/.env.example

```
@@ -5,3 +5,20 @@ OTEL_SERVICE_NAME="<service_name>"
 
 # database
 DB_CONNECTION="Server=localhost,14333;Database=AppDb;User ID=sa;Password=LocalDev123!;Encrypt=False;TrustServerCertificate=True;"
+
+# smtp delivery
+Smtp__Host="<smtp_host>"
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The Smtp__Host key should be in uppercase

(LowercaseKey)